SerialPrograms/Source/PokemonSV/Programs/Eggs/PokemonSV_EggAutonomous.cpp

```diff
@@ -324,7 +324,8 @@ void EggAutonomous::hatch_eggs_full_routine(SingleSwitchProgramEnvironment& env,
     // The loop to hatch batches of eggs.
     // Each batch consists of at most five eggs.
     // There are at most six batches of eggs in a box.
-    uint8_t next_egg_column = 0; // next egg column in box
+    uint8_t next_egg_column = 1; // next egg column in box
+    // starting at 1 because picnic_party_to_hatch_party already loaded and hatched the first batch.

```

Comment:
A mistake here. picnic_party_to_hatch_party only load the first column into the party.
The first column is not hatched yet. It is hatched during the first iteration at line 336 hatch_eggs_at_zero_gate